Math_Game/math_game.py

```diff
@@ -30,14 +29,11 @@ def operation(self, a, b):
 
 class divide(MyMath):
     def operation(self, a, b):
-        result = a/b
-
-        if result.is_integer():
-            formatted_result = int(result)
+        # floor division
+        if a < b:
```

Comment:
This is a HUGE bug. Never mess with the API. Your API is a contract which promises that all operations will return `a operation b`. Add custom logic like this outside your API (in your main() in this case).

```
if options[selection] == "divide":
    if a < b:
        a, b = b, a
    print("\nTo simplify division, the answer will be a whole number i.e. how many times can the higher number divide into the smaller number.")
```

Reply:
to clarify about the API issue: 

Your MyMath class can be made into its own module and reused by other programs that need a succinct set of basic math functions. That's a good coding practice.

As part of the game rules you might have decided that on April Fools day, all the correct answers should have a random integer added to them as a joke on the player. That's a perfectly fine rule for for the game to implement inside the game code.  But if you implement that logic inside the MyMath API itself, the joke code will break every other app that is using the module. Those applications are expecting a basic set of math functions.. Since the joke code only triggers on April fools day, the bug will never be reproducible. 

Let's say you used the MyMath API in a stock tracking tool that uses the module's API to calculate profit/loss and determine whether it should sell a stock for trading outside a trading range. On April Fools day it could accidentally trigger a sale of your whole stock portfolio. April Fools!